source/computer/macos-setup.rst

```diff
@@ -258,6 +258,11 @@ macOS 系统自带的文本编辑器只具有最基本的文本编辑功能。
 
     $ brew install --cask visual-studio-code
 
+`Typora <https://typora.io/>`__ 是一款\ **轻便简洁**\ 的 Markdown 编辑器，支持即时渲染技术，
+适合日常临时编写小型 Markdown 文件::
+
+    $ brew install --cask typora
+
```

Comment:
VSCode can already do the same thing. Do we need another editor?

Reply:
I think so. It's not good to use vscode to editor some small markdown files.
It is better to have a very light markdown editor.

Reply:
> It's not good to use vscode to editor some small markdown files.

Why do you think so?

Reply:
Otherwise, I have to open vscode to editor a small markdown file. It is a little slow to open vscode than typora.
It depends that whether you like a lighter markdown editor.

Reply:
> Otherwise, I have to open vscode to editor a small markdown file. It is a little slow to open vscode than typora.
> It depends that whether you like a lighter markdown editor.

I agree that vscode is a little slow in starting up. I usually always keep the vscode open and opening other files will be much faster.

Reply:
Or we may leave `Typora` here and add a note it is very fast to start up? Let the reader to choose if you want suah a lighter markdown editor?